packages/faceted-search/i18n-extract/0.9.0/tui-faceted-search.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
gitignore?

Reply:
Same for `packages/faceted-search/i18n-extract/i18n.zip`, do we need to versionning the /i18n-extract folder?